Library/Formula/fpp.rb

```diff
@@ -0,0 +1,20 @@
+class Fpp < Formula
+  homepage "https://facebook.github.io/PathPicker/"
+  url "https://facebook.github.io/PathPicker/dist/fpp.0.5.3.tar.gz"
+  sha256 "94b77ef10a128a694f6302ce578a2d07a3fd2892299d341b22be9496abd7277d"
+  head "https://github.com/facebook/pathpicker.git"
```

Comment:
yeah we have to keep it private for now since moving it to public creates a newsfeed event for all our followers, but ironically the `gh-pages` website is totally open.

but yeah i double checked the head link -- works if you are part of the FB team